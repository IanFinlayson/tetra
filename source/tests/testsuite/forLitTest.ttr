def main():
   sum = 0
   print("Just created sum and sum = ", sum, "\n")
   for num in [1 ... 10]:
      print("In for loop and num = ", num, "\n")
      sum += num
      print("Just added num to sum and sum = ", sum, "\n")
   print("Just exited for loop and sum = ", sum, "\n")
