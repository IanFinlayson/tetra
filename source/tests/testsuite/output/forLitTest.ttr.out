Just created sum and sum = 0
In for loop and num = 1
Just added num to sum and sum = 1
In for loop and num = 2
Just added num to sum and sum = 3
In for loop and num = 3
Just added num to sum and sum = 6
In for loop and num = 4
Just added num to sum and sum = 10
In for loop and num = 5
Just added num to sum and sum = 15
In for loop and num = 6
Just added num to sum and sum = 21
In for loop and num = 7
Just added num to sum and sum = 28
In for loop and num = 8
Just added num to sum and sum = 36
In for loop and num = 9
Just added num to sum and sum = 45
In for loop and num = 10
Just added num to sum and sum = 55
Just exited for loop and sum = 55

